step/step.go

```diff
@@ -134,8 +134,14 @@ func (testRunner InstrumentedTestRunner) Run(config Config) error {
 	}
 
 	// `adb` does not return an error exit code when tests fail, so we parse the test log instead
-	if strings.Contains(outputBuffer.String(), "FAILURES!!!") {
-		return fmt.Errorf("test run contained at least one test failure")
+	patterns := []string {
+		"FAILURES!!!",
```

Comment:
I do not know why but I smiled at this line. Like one exclamation point was not enough, lets add three. :D

Reply:
Poor man's log severity level 😛